src/preamble_minimal.js

```diff
@@ -212,9 +212,9 @@ function callRuntimeCallbacks(callbacks) {
     var func = callback.func;
     if (typeof func === 'number') {
       if (callback.arg === undefined) {
-        dynCall_v(func);
+        Module['dynCall_v'](func);
```

Comment:
Is this needed because `dynCall_v` only exists if there is a use of it? We could know that at compile time to shrink code size here, I think. But can leave that separately from this.

Reply:
Hmm, this is against the intent of MINIMAL_RUNTIME, which wants to deprecate internal use of `Module` object. In MINIMAL_RUNTIME we do not (want to) export `dynCall_v` out to Module like this. Can we instead do

```js
    if (typeof func === 'number') {
      if (callback.arg === undefined) {
        if (typeof(dynCall_v !== 'undefined') dynCall_v(func);
      }
```
or force-include `dynCall_v` and `dynCall_vi` in the build if EXIT_RUNTIME is present. (although that is somewhat annoying as well)

If neither of the above work and we should depend on `Module` to make the compiler realize it needs to emit dynCall sigs for `v` and `vi`, then let's add at minimum a `/* TODO: Replace access of Module['dynCall_v'] with a direct access to dynCall_v(). */`

Reply:
Oh, I didn't know about that objective regarding the Module object.     I'll see what I can do.

Reply:
It looks like it was just a closure compiler issue.   Adding some suppressions fixed it.

Reply:
OK I figured it out.. they belong in `src/closure-externs/closure-externs.js`

PTAL.